cfi_csrs.adoc

```diff
@@ -16,41 +16,32 @@ This chapter specifies the CSR state of the Zicfisslp extension.
   {bits:  1, name: 'CBZE'},
   {bits: 51, name: 'WPRI'},
   {bits:  1, name: 'SFCFIE'},
-  {bits:  1, name: 'CFIE'},
+  {bits:  1, name: 'SBCFIE'},
   {bits:  1, name: 'HADE'},
   {bits:  1, name: 'PBMTE'},
   {bits:  1, name: 'STCE'},
 ], config:{lanes: 4, hspace:1024}}
 ....
 
-The `CFIE` field (bit 60) controls whether Zicfisslp extension is available for
-use in modes less privileged than M. When `CFIE` is 1, the `SFCFIE` field
-(bit 59) activates forward-edge CFI at S-mode.
+Zicfiss extension introduces the `SBCFIE` field (bit 60) in `menvcfg`. When
+`SBCFIE` field is 1, the Zicfiss extension is active in S-mode. When `SBCFIE`
+field is 0, the Zicfiss extension is not active in S-mode and the following
+rules apply to privilege modes less than M.
 
-When `menvcfg.CFIE` bit is 0, the following rules apply to privilege modes less
-privileged than M:
-
-* Attempts to access the `ssp` CSR result in an illegal-instruction exception.
-* Shadow stack instructions revert to the Zimop/Zcmop defined behavior.
-* The `lpad` instruction executes as a no-op.
-* The `UBCFIE`, `UFCFIE`, and `SPELP` fields in `sstatus` are read-only zero.
-* The `CFIE` field in `henvcfg` is read-only zero.
+* Attempts to access the `ssp` CSR raise an illegal-instruction exception.
+* The 32-bit Zicfiss instructions revert to their Zimop defined behavior.
+* The 16-bit Zicfiss instructions revert to their Zcmop defined behavior.
 * The `pte.xwr=010b` encoding in S-stage page tables is reserved.
```

Comment:
The description of `pte.xwr=010b` in "Virtual-Memory system extension for Shadow Stack" still references `menvcfg.CFIE`.

Are shadow stack instructions allowed to access memory if `[v]satp.mode` = `Bare`?

Reply:
shadow stack instructions are allowed to access memory if when virtual memory system mode is `Bare`.
Will fix the "CFIE->SBCFIE"